### DISCUSSION TITLE
Azure Functions - TimerTrigger - Get Next Scheduled Occurence?

### DISCUSSION THREAD

**Discussion by dfaivre** [Azure/azure-functions-dotnet-worker#2204] [Feedback]
Is there a way to get the next scheduled occurence of an Azure Timer function? This issue seems to say that `TimerInfo` should have a schedule property, but I'm not seeing it in .NET 8.

https://github.com/Azure/azure-functions-dotnet-worker/issues/954#issuecomment-1238433441

```
public static void CronJob([TimerTrigger("*/15 * * * * *")] TimerInfo timerInfo)
{
    var schedule = timerInfo.Schedule as CronSchedule;
    var next = schedule.GetNextOccurrence(DateTime.UtcNow);

    . . .
}
```

Is the `GetNextOccurrence` ability just gone now? Is there any workaround?

**Comment by dfaivre** [comment]
Cross posted: https://stackoverflow.com/questions/77802173/azure-functions-timertrigger-get-next-scheduled-occurence

**Comment by fabiocav** [comment]
Hey @dfaivre , I've moved this to an issue so we can track this in our backlog. 

Thank you!
